tests/test_migrate.py

```diff
@@ -104,49 +104,57 @@ def test_migrate_transform_job_id(migrator: Migrator):
     assert migrate_batch.transform_job_id == "xxx"
 
 
-def test_migrate_batch_records_success(migrator: Migrator, md_fixtures: list[Fixture]):
-    batch, _ = get_transform_results("noop", migrator)
-    migrate_batch = migrator.migrate(batch, overwrite=False, group=None)
+def test_migrate_batch_records_success(
+    migrator: Migrator, md_fixtures: list[Fixture], group_fixture: int
+):
+    batch, _ = get_transform_results("change-language", migrator)
+    migrate_batch = migrator.migrate(batch, overwrite=False, group=group_fixture)
+    assert len(migrate_batch.successes())
```

Comment:
```suggestion
    len(migrate_batch.successes()) == len(batch.successes())
```
or at least (for readability)
```suggestion
    len(migrate_batch.successes()) > 0
```